client/service-container-chain/src/data_preservers.rs

```diff
@@ -52,11 +54,15 @@ pub async fn task_watch_assignment(spawner: impl Spawner, profile_id: ProfileId)
                 .data_preserver_active_assignment(hash, profile_id)
                 .await?;
 
-            log::info!("Assignement for block {hash}: {new_assignment:?}");
+            if current_assignment == new_assignment {
+                continue;
+            }
+
+            log::info!(
```

Comment:
same thing, or maybe even create a log target called data_preservers

Reply:
This only logs when the state changes, and the other log only once when the node starts, it's ok for me.

Reply:
I react on this log in the typescript tests. I can add a log target however